reya-rust-sdk/src/http_provider.rs

```diff
@@ -283,7 +283,14 @@ impl HttpProvider {
                 //     price_limit,   // price limit is the slippage tolerance,we can set it to max uint or zero for now depending on the direction of the trade
                 // }// endcoded
                 let trigger_price = batch_order.stop_price;
-                let bytes = (batch_order.is_long, trigger_price, batch_order.price_limit)
+                let multiplier: I256 = I256::from_dec_str("10").unwrap().pow(U256::from(18));
```

Comment:
question to understand: why I256 if the price should be unsigned?

Reply:
trigger price is unsigned?

Reply:
it is yeah, lemme check

Reply:
https://github.com/Reya-Labs/reya-rust-sdks-monorepo/pull/8/commits/74dabd9cbae5446797c8eca5732dbc0ddf29e635